rasa/nlu/model.py

```diff
@@ -389,10 +389,11 @@ def parse(
             output["text"] = ""
             return output
 
+        timestamp = str(int(time.timestamp())) if time else None
         data = self.default_output_attributes()
         data[TEXT] = text
 
-        message = Message(data=data, time=time)
+        message = Message(data=data, time=timestamp)
```

Comment:
seems we should rather pass an `int` here instead of `Text`. The only usages of `time` I found, cast to an `int` 🙈

Reply:
I don't have enough knowledge about this to make the decision myself, but don't you think it would be a breaking change that could impact users?

Reply:
I think it's safe to change. `UserUttered` uses a separate `float` timestamp anyway which means users shouldn't see this timestamp ever.